src/app/(app)/globals.css

```
@@ -0,0 +1,96 @@
+@import "tailwindcss";
+@import "tw-animate-css";
+@plugin "daisyui" {
+  themes: all;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)